.gitignore

```diff
@@ -93,3 +93,4 @@ package-lock.json
 
 CHANGELOG.md
 coursier
+/bin/
```

Comment:
I'm just curious, what do you have there, why do you need to ignore it in every project?